src/NexusMods.Collections/InstallCollectionJob.cs

```diff
@@ -196,6 +266,77 @@ await Parallel.ForEachAsync(archive.Children, async (child, token) =>
         return result.Remap(group);
     }
 
+    /// <summary>
+    /// This will go through and generate all the patch files for the given archive based on the mod's patches.
+    /// </summary>
+    private async Task CreatePatches(Mod modInfo, LibraryArchive.ReadOnly modArchive, LibraryArchive.ReadOnly collectionArchive, 
```

Comment:
Could this function please be renamed?
The description and title is misleading.

When it comes to binary diffing the common terminology is:

- **Input:** (Source/Original) file
- **Diff:** (Diff/Patch) File
- **Output:** Output/Target File

The existing name `CreatePatches` implies that we're creating the actual `bsdiff` patches themselves; this is not the case; we're applying an existing `bsdiff` patch, and storing the resulting patched files in the `FileStore`.

-------------

Misc note: The design sucks here.

> and no one ever thought to provide a link between MD5 hashes and patches. And the patch's hash is a CRC32, because why not?

I feel your pain; this sucks.